src/library/Uno.Material/Styles/Application/AnimationConstants.xaml

```diff
@@ -1,10 +1,16 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
 
-	<CubicEase x:Key="Material_EaseInOutFunction"
+	<CubicEase x:Key="MaterialEaseInOutFunction"
 			   EasingMode="EaseInOut" />
 
-	<CubicEase x:Key="Material_EaseOutFunction"
+	<CubicEase x:Key="MaterialEaseOutFunction"
 			   EasingMode="EaseOut" />
 
+	<Duration x:Key="MaterialAnimationDuration">0:0:0.25</Duration>
+
+	<Duration x:Key="MaterialSnackbarAnimationDuration">0:0:0.4</Duration>
+	<Duration x:Key="MaterialTextBoxAnimationDuration">0:0:0.25</Duration>
```

Comment:
This value is the same as `MaterialAnimationDuration`, did you forget to remove it after refactoring?

Reply:
No, Chunk asked that textbox had their own variables for animation duration for the translate transform of the placeholder text